app/controllers/posts_cotroller.rb

```diff
@@ -1,5 +1,10 @@
 class PostsController < ApplicationController
```

Comment:
- [ ] Kindly update the name of this file to `posts_controller.rb` instead of `posts_cotroller.rb`. I get an error when I go to any post-controller action.
![image](https://user-images.githubusercontent.com/24703171/200761396-f82b9024-7d91-46c6-bd5e-99a9b68a8662.png)